examples/bore_field_thermal_resistance.py

```diff
@@ -6,15 +6,18 @@
     rates.
 
 """
-import matplotlib.pyplot as plt
+try:
+    import matplotlib.pyplot as plt
+except ModuleNotFoundError:
+    pass
+
```

Comment:
@myoldmopar, can you take a look here before I push it all the way through?

Reply:
Might consider storing a variable to indicate whether it has been properly imported or not.

Reply:
```suggestion
try:
    import matplotlib.pyplot as plt
    plotting_enabled = True
except ModuleNotFoundError:
    plotting_enabled = False

```